rotkehlchen/db/upgrades/v31_v32.py

```diff
@@ -155,6 +155,13 @@ def _update_history_entries_from_kraken(cursor: 'Cursor') -> None:
         )
 
 
+def _update_settings_name_for_selected_binance_markets(cursor: 'Cursor') -> None:
+    cursor.execute("""
+    UPDATE user_credentials_mappings SET setting_name = binance_selected_trade_pairs
+    WHERE setting_name = PAIRS
```

Comment:
Did you even try to run this or test it? Tests fail since you are not using a string literal.

```suggestion
    UPDATE user_credentials_mappings SET setting_name ="binance_selected_trade_pairs"
    WHERE setting_name="PAIRS"
```

Also please use the string literal directly from db/constants.py